tests/baselines/reference/checkJsxChildrenProperty2.errors.txt

```diff
@@ -3,19 +3,19 @@ tests/cases/conformance/jsx/file.tsx(17,11): error TS2710: 'children' are specif
 tests/cases/conformance/jsx/file.tsx(31,6): error TS2322: Type '{ children: (Element | ((name: string) => Element))[]; a: number; b: string; }' is not assignable to type 'Prop'.
   Types of property 'children' are incompatible.
     Type '(Element | ((name: string) => Element))[]' is not assignable to type 'string | Element'.
-      Type '(Element | ((name: string) => Element))[]' is missing the following properties from type 'Element': type, props
+      Type '(Element | ((name: string) => Element))[]' is not assignable to type 'string'.
 tests/cases/conformance/jsx/file.tsx(37,6): error TS2322: Type '{ children: (number | Element)[]; a: number; b: string; }' is not assignable to type 'Prop'.
   Types of property 'children' are incompatible.
     Type '(number | Element)[]' is not assignable to type 'string | Element'.
-      Type '(number | Element)[]' is missing the following properties from type 'Element': type, props
+      Type '(number | Element)[]' is not assignable to type 'string'.
 tests/cases/conformance/jsx/file.tsx(43,6): error TS2322: Type '{ children: (string | Element)[]; a: number; b: string; }' is not assignable to type 'Prop'.
   Types of property 'children' are incompatible.
     Type '(string | Element)[]' is not assignable to type 'string | Element'.
-      Type '(string | Element)[]' is missing the following properties from type 'Element': type, props
+      Type '(string | Element)[]' is not assignable to type 'string'.
 tests/cases/conformance/jsx/file.tsx(49,6): error TS2322: Type '{ children: Element[]; a: number; b: string; }' is not assignable to type 'Prop'.
   Types of property 'children' are incompatible.
     Type 'Element[]' is not assignable to type 'string | Element'.
-      Type 'Element[]' is missing the following properties from type 'Element': type, props
+      Type 'Element[]' is not assignable to type 'string'.

```

Comment:
Regression :(

Reply:
It's probably better this way, for now, until the element -> argument case has a special handler.